diff --git a/src/Vulkan/VkConfig.hpp b/src/Vulkan/VkConfig.hpp
index cc4e3b9..3c16da5 100644
--- a/src/Vulkan/VkConfig.hpp
+++ b/src/Vulkan/VkConfig.hpp
@@ -79,7 +79,7 @@ static_assert(MAX_IMAGE_LEVELS_2D <= sw::MIPMAP_LEVELS);
 static_assert(MAX_IMAGE_LEVELS_3D <= sw::MIPMAP_LEVELS);
 static_assert(MAX_IMAGE_LEVELS_CUBE <= sw::MIPMAP_LEVELS);
 
-constexpr uint32_t MAX_BOUND_DESCRIPTOR_SETS = 4;
+constexpr uint32_t MAX_BOUND_DESCRIPTOR_SETS = 8;
 constexpr uint32_t MAX_VERTEX_INPUT_BINDINGS = 16;
 constexpr uint32_t MAX_PUSH_CONSTANT_SIZE = 128;
 constexpr uint32_t MAX_UPDATE_AFTER_BIND_DESCRIPTORS = 500000;
